### PR TITLE
Add ES

### PR DESCRIPTION
**Pull request by ilanolkies**
Part of #28

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -28,6 +28,26 @@ const resources = {
...
     translation: {
     }
+  },
+  es: {
+    translation: {
+      'Connecting to server': 'Conectandose al servidor',
+      'Connect your wallet': 'Conecte su billetera',
+      'No wallets found': 'No se ha encontrado ninguna billetera',
+      'No wallet?': 'No tiene billetera?',
+      'Get one here!': 'Descargue una aqui!',
```
Comment (lucachaco) [inline src/i18n.js:38]: aquí

File: src/i18n.js [hunk 2]
```diff
@@ -28,6 +28,26 @@ const resources = {
...
+      'Would you like to give us access to info in your data vault?': 'Nos daria acceso a la información requerida de su Data Vault?',
```
Comment (lucachaco) [inline src/i18n.js:39]: daría

File: src/i18n.js [hunk 3]
```diff
@@ -28,6 +28,26 @@ const resources = {
...
+      'Get the information from your Data Vault that you want to share with the following service': 'Descargue la informacion que desea compartir con el servicio',
```
Comment (lucachaco) [inline src/i18n.js:40]: información

File: src/i18n.js [hunk 4]
```diff
@@ -28,6 +28,26 @@ const resources = {
...
+      'Access Data Vault': 'Acceda al Data Vault',
+      'Connecting to the DataVault': 'Conectandose al Data Vault',
+      'DataVault Error': 'Error en el Data Vault',
+      'Select information to share': 'Seleccione la información que desea compartir',
+      'Sharing your information is optional. It will only be shared with': 'Compartir su información es opcional. Solo se compartira con',
```
Comment (lucachaco) [inline src/i18n.js:45]: sólo

File: src/i18n.js [hunk 5]
```diff
@@ -28,6 +28,26 @@ const resources = {
...
+      Confirm: 'Confirmar',
+      'Use this Identity?': 'Usar esta identidad?',
+      'Confirm Identity': 'Confirmar Identity',
```
Comment (lucachaco) [inline src/i18n.js:48]: identidad

File: src/i18n.js [hunk 6]
```diff
@@ -28,6 +28,26 @@ const resources = {
...
+      'Confirming Identity': 'Confirmando Identity'
```
Comment (lucachaco) [inline src/i18n.js:49]: identidad

**Review by lucachaco — CHANGES_REQUESTED** [review]
Just small spelling errors

**Comment by jessgusclark** [comment]
Rebased from `develop`, changed the base to `develop` and fixed suggestions from @lucachaco

**Review by lucachaco — APPROVED** [review]

**Comment by jessgusclark** [comment]
GH had an issue last night and the CI didn't run. I have added empty commit to trigger it to run again.

**Pull request merged**